Microsoft.VCRTForwarders.140.targets

```diff
@@ -4,34 +4,91 @@
   <PropertyGroup>
     <!-- Output folder for MSBuildForUnity -->
     <VCRTForwarders-PackageDestinationFolder>$(MSBuildThisFileName)</VCRTForwarders-PackageDestinationFolder>
+    <VCRTForwarders-IncludeDebugCRT Condition="'$(VCRTForwarders-IncludeDebugCRT)' == '' And '$(Configuration)' == 'Debug'">true</VCRTForwarders-IncludeDebugCRT>
```

Comment:
cool trick:
Condition="'$(VCRTForwarders-IncludeDebugCRT)$(Configuration)' == 'Debug'"

Reply:
That is definitely cool and makes sense why it works.  Will use it.